src/style.css

```diff
@@ -0,0 +1,97 @@
+#x {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
```

Comment:
Set font styles to html tag, other elements will inherit (except placeholders, you have to use `inherit` for them)